src/DataFixtures/ORM/LoadSummaryData.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+namespace AppBundle\DataFixtures\ORM;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\MemberSummary\JobExperience;
+use AppBundle\Entity\MemberSummary\Language;
+use AppBundle\Entity\MemberSummary\MissionType;
+use AppBundle\Entity\MemberSummary\Skill;
+use AppBundle\Entity\MemberSummary\Training;
+use AppBundle\Summary\Contract;
+use AppBundle\Summary\Contribution;
+use AppBundle\Summary\JobDuration;
+use AppBundle\Summary\JobLocation;
+use AppBundle\Summary\SummaryFactory;
+use Cocur\Slugify\Slugify;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class LoadSummaryData implements FixtureInterface
+{
+    public function load(ObjectManager $manager)
+    {
+        $summaryFactory = $this->getSummaryFactory();
+
+        $summary1 = $summaryFactory->createFromArray([
+            'adherent' => $manager->getRepository(Adherent::class)->findByUuid(LoadAdherentData::ADHERENT_1_UUID),
+            'slug' => 'summary-michelle-dufour',
+            'availability' => JobDuration::PART_TIME,
+            'contactEmail' => 'michelle.dufour.2@example.ch',
+            'contributionWish' => Contribution::VOLUNTEER,
+            'jobLocation' => JobLocation::ON_REMOTE,
+            'motivation' => 'C\'est mon secret',
+            'professionalSynopsis' => '30 ans de travail dans le domaine scientifique',
+            'currentProfession' => 'Bio-informaticien',
+            'websiteUrl' => 'https://michelle-dufour-fake.en-marche-dev.fr',
+            'viadeoUrl' => 'https://fr.viadeo.com/michelle-dufour-fake',
+            'facebookUr' => 'https://www.facebook.com/michelle-dufour-fake',
+            'twitterNickname' => 'https://twitter.com/michelle-dufour-fake',
+            'missionWishes' => $manager->getRepository(MissionType::class)->findBy(['name' => 'Missions de bénévolat']),
+        ]);
+        $manager->persist($summary1);
+
+        // Experiances
+        $experiance11 = new JobExperience();
```

Comment:
Hello,

experience is mispelled. But maybe there is a reason for that.

Cheers